src/components/common/AlertModal/index.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ReactElement } from 'react'
+import { Styled } from './styled'
+import type { AlertModalProps } from './types'
+
+const AlertModal = ({
+  isOpen = true,
+  isCheckShape = false,
+  title,
+  subTitle,
+  buttonContents,
+  onClose,
+  onClick
```

Comment:
```suggestion
  onConfirm
```
변수명에 혼란이 있을 것 같아 onConfirm도 괜찮을 것 같습니다!

Reply:
오 그렇네요! 
더 확실하게 onClickComfirmButton으로 하겠습니다!

Reply:
완료!
[링크](https://github.com/price-offer/offer-fe/pull/108/commits/2f6a4f180e5cc19c7c09ed82b4805a718940c178#diff-08768223cb51d41ee8726f9e11b78640fecff8209b3c03c0986f90775f3fb7f8)